tests/unit/browser/webengine/test_darkmode.py

```diff
@@ -32,6 +32,19 @@ def patch_backend(monkeypatch):
     monkeypatch.setattr(objects, 'backend', usertypes.Backend.QtWebEngine)
 
 
+@pytest.mark.parametrize('enabled, expected', [
+    # Disabled or nothing set
+    (False, []),
+
+    # Enabled in configuration
+    (True, [("preferredColorScheme", "1")]),
+])
+@utils.qt515
```

Comment:
Why not make this `qt514` and parametrize the Qt version as well, so that we can make sure it doesn't get added on Qt < 5.15.2?

Reply:
Good idea. I'll do that.